### PR TITLE
merge repos

### PR DESCRIPTION
**Pull request by mcallegari10**
## Description

Merge training repos. The tasks done are up to the React Router.

## Screenshots

image

image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import './index.css';
+
+ReactDOM.render(
+  <HashRouter>
+    <div>
+      <Route exact path='/' component={ App } />
+      <Route exact path='/dashboard' component={ Dashboard } />
+      <Route exact path='/book/:bookId' component={ BookDetail } />
+    </div>
+  </HashRouter>,
```
Comment (pabloferro) [inline src/index.js:17]: Move this to a routes component?

File: src/screens/App/screens/BookDetail/components/BookInfo/index.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import './BookInfo.css';
+
+const lorem = 'Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.';
+
+function BookInfo(props) {
```
Comment (pabloferro) [inline src/screens/App/screens/BookDetail/components/BookInfo/index.js]: Use destructuring

File: src/screens/App/screens/BookDetail/components/Comments/components/MakeComment/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import userDefault from '../../../../../../../../assets/user.png';
+import './MakeComment.css';
+
+const MakeComment = props =>
```
Comment (pabloferro) [inline src/screens/App/screens/BookDetail/components/Comments/components/MakeComment/index.js]: use function and destructuring to be consistent.

File: src/screens/App/screens/BookDetail/components/Comments/components/MakeComment/index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import userDefault from '../../../../../../../../assets/user.png';
+import './MakeComment.css';
+
+const MakeComment = props => 
+  <div className='new-comment-container'>
+    <img className='profile-picture' src={ userDefault } alt='User'/>
```
Comment (pabloferro) [inline src/screens/App/screens/BookDetail/components/Comments/components/MakeComment/index.js]: Maybe receive avatar from props and use default props to set userDefault?

File: src/screens/App/screens/BookDetail/components/Comments/components/ShowComment/index.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import userDefault from '../../../../../../../../assets/user.png';
+import './ShowComment.css';
+
+const ShowComment = props =>
+  <div className='comment'>
+    <img className='profile-picture' src={ userDefault } alt='User'/>
```
Comment (pabloferro) [inline src/screens/App/screens/BookDetail/components/Comments/components/ShowComment/index.js]: This is repeated, maybe extract a component?

File: src/screens/App/screens/BookDetail/components/Comments/components/ShowComment/index.js [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+    <div className='comment-info'>
+      <span className='user-name'>User Name</span>
+      <span className='comment-date'>xx/xx/xx</span>
```
Comment (pabloferro) [inline src/screens/App/screens/BookDetail/components/Comments/components/ShowComment/index.js]: Props?

File: src/screens/App/screens/Dashboard/index.js [hunk 7]
```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import DashboardScreen from './screen';
```
Comment (pabloferro) [inline src/screens/App/screens/Dashboard/index.js]: We are naming the presentational components Layout.

File: src/screens/App/screens/Dashboard/index.js [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+    this.setState({ filterAttr: event.target.value });
+  }
+
+  filterBooks() {
+    return books.filter((book) => {
+      return book[this.state.filterAttr].toLowerCase().includes(this.state.filterValue.toLowerCase());
+    });
+  }
+
+  changeBooks = (event) => {
```
Comment (pabloferro) [inline src/screens/App/screens/Dashboard/index.js]: handleFilter?

**Review by pabloferro — CHANGES_REQUESTED** [review]

File: src/screens/App/screens/BookDetail/components/BookInfo/index.js (hunk 2 above)
Comment (mcallegari10) [reply src/screens/App/screens/BookDetail/components/BookInfo/index.js]: Already changed in #2

File: src/screens/App/screens/BookDetail/components/Comments/components/MakeComment/index.js (hunk 3 above)
Comment (mcallegari10) [reply src/screens/App/screens/BookDetail/components/Comments/components/MakeComment/index.js]: Changed in #2

File: src/screens/App/screens/Dashboard/index.js (hunk 7 above)
Comment (mcallegari10) [reply src/screens/App/screens/Dashboard/index.js]: Changed in #2

**Review by pabloferro — APPROVED** [review]

**Pull request merged**